### DISCUSSION TITLE
Sous-titre de vtt vers srt

### DISCUSSION THREAD

**Discussion by Philipeau** [NickvisionApps/Parabolic#939] [Ideas]
Magnifique version ! Deux idées :
- Est-il possible d'ajouter l'option de conversion des sous-titres en srt (elle marche bien maintenant avec --convert-subtitles srt) ?
- la possibilité d'avoir deux audios et une sortie en mkv ?
Bravo pour votre travail vraiment très pratique !
--------------------------------------
Wonderful version! Two ideas:
-    Is it possible to add the option to convert subtitles to srt (it works well now with --convert-subtitles srt)?
-    the possibility of having two audios and an output in mkv?
Bravo for your really practical work!

**Comment by nlogozzo** [comment]
> Is it possible to add the option to convert subtitles to srt (it works well now with --convert-subtitles srt)?

See #932 . It will be available in the next version.

> the possibility of having two audios and an output in mkv?

See #678 . It will not be available in the next version but hopefully soon!

> Bravo for your really practical work!

Thank you!
